ReactWindows/ReactNative.Shared/UIManager/AccessibilityTrait.cs

```diff
@@ -0,0 +1,64 @@
+namespace ReactNative.UIManager
+{
+    /// <summary>
+    /// NOTE: This enum is organized based on priority of these traits (0 is the lowest),
+    /// which can be assigned to an accessible object. On native, all traits are combined as
+    /// a list. On desktop, trait with the maximum value is picked. Whenever you are adding
+    /// a new trait add it in the right priority order in the list.
+    ///
+    /// This is copied from ReactXP definition of AccessibilityTrait enum.
```

Comment:
>This is copied from ReactXP definition of AccessibilityTrait enum. [](start = 8, length = 66)

I don't think this is important to note. react-native-windows is a layer below ReactXP so react-native-windows should be the one defining the contract. Not ReactXP.

Reply:
agree. removed.